### DISCUSSION TITLE
Can we remove such warning in pdf generated through latexpdf ?

### DISCUSSION THREAD

**Discussion by arafune** [sphinx-doc/sphinx#12799] [Q&A] [answered]
In the pdf file generated by Sphinx with latexpdf,  the following warning appears:

<img width="1036" alt="スクリーンショット 2024-08-18 12 17 28" src="https://github.com/user-attachments/assets/5124e60a-fd79-40ac-8001-a6e62a28478d">

Are there anyway who know the way to suppress this message ?

**Comment by mgeier** [comment, marked as answer]
These warnings are generated by the `nbsphinx` extension, an issue has already been created: https://github.com/spatialaudio/nbsphinx/issues/811
